spec/requests/api/v1/contacts/create_spec.rb

```diff
@@ -0,0 +1,185 @@
+require "rails_helper"
+
+describe "Contacts Controller", type: :request do
+  describe "#create action - Happy Paths" do
+		before(:each) do
+			@user1 = User.create!(name: "Me", email: "happy@gmail.com", password: "reallyGoodPass")
+			user_params = { email: "happy@gmail.com", password: "reallyGoodPass" }
```

Comment:
Maybe it's the way gh is rendering your changes, but the indentation looks exaggerated in my reader.

Reply:
We do have 2 space indentation on, not sure why it is looking like this.

Reply:
See above--issue was with my Github settings.